File: SDKs/letmc.com/customer/v2-customer/python/src/sdk/models/shared/landlordmaintenancepreferencemodel.py
from dataclasses import dataclass, field
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclass
class LandlordMaintenancePreferenceModel:
    r"""LandlordMaintenancePreferenceModel
    Maintenance Preference
    """
    
    name: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Name') }})
    notes: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Notes') }})
    type: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Type') }})
    

File: SDKs/letmc.com/customer/v2-customer/typescript/src/sdk/models/shared/landlordmaintenancepreferencemodel.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// LandlordMaintenancePreferenceModel
/** 
 * Maintenance Preference
**/
export class LandlordMaintenancePreferenceModel extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=Name" })
  name?: string;

  @SpeakeasyMetadata({ data: "json, name=Notes" })
  notes?: string;

  @SpeakeasyMetadata({ data: "json, name=Type" })
  type?: string;
}
